app/api/views/auth.py

```diff
@@ -113,6 +115,15 @@ def auth_register():
         ).send()
         return jsonify(error=f"cannot use {email} as personal inbox"), 400
 
+    if check_if_abuser_email(email):
+        # TODO: do I need to call RegisterEvent in here?
+        emit_abuser_audit_log(
+            None,
+            AbuserAuditLogAction.CreateUser,
+            "An attempt to register a previously banned email",
+        )
+        return jsonify(error=f"cannot use {email} as it was previously banned"), 400
+
```

Comment:
```suggestion
    if check_if_abuser_email(email):
        LOG.warn("User with email {email} that was marked as abuser tried to register again")
        return jsonify(error=f"cannot use {email} as an email to register"), 400
```
The less info we give abusers the better :)

Reply:
We have to also do it in `app/auth/views/register.py` line 74